issues.go

```diff
@@ -81,48 +81,51 @@ type IssueLinks struct {
 	Project    string `json:"project"`
 }
 
+// IssueTasksCompletionStatus represents tasks of the issue.
+type IssueTasksCompletionStatus struct {
+	Count          int `json:"count"`
+	CompletedCount int `json:"completed_count"`
+}
```

Comment:
Would `TaskCompletionStatus` maybe be enough? En can you place it blow the main `Issue` struct?

Reply:
Other, `IssueReferences`, `IssueCloser`, etc are above `Issue` and use "Issue" prefix. I'll move/rename it anyway ?

Reply:
The rest of the PR looks good now 👍🏻  Guess the main question for this remaining issue is what the usecase it to make a named struct for it? If there isn't a good reason, why not keep it as an inline struct instead?

Reply:
I'have create this `IssueTasksCompletionStatus` struct because, it exists on `TodoTarget`:

![image](https://user-images.githubusercontent.com/30216166/119237643-28b13280-bb3e-11eb-8e37-c0d97d636cd4.png)